frontend/src/components/GlobalStyle.js

```diff
@@ -0,0 +1,17 @@
+import { createGlobalStyle } from 'styled-components'
+
+export default createGlobalStyle`
+  html, body {
+    font-size: 112.5%;
+  }
+  
+  :root {
+    --color-deep-space-sparkle: #526768;
+    --color-polished-pine: #6A9882;
+    --color-turquoise-green: #94CBB1;
+    --color-cambridge-blue: #A7C1AF;
+    --color-alabaster: #E3EADB;
+    --color-cultured: #FAFAFA;
+    --color-silver: #C4C4C4; 
```

Comment:
I wish github would preview the colors like intellij does, I am gonna trust your aesthetic sense anyway.

Reply:
yeah that would be great 🙈
otherwise color scheme can be found here:
https://coolors.co/526768-6a9882-94cbb1-a7c1af-e3eadb-fafafa-c4c4c4